vendor/ctpl_stl.h

```diff
@@ -138,8 +159,8 @@ namespace ctpl {
                 this->cv.notify_all();  // stop all waiting threads
             }
             for (int i = 0; i < static_cast<int>(this->threads.size()); ++i) {  // wait for the computing threads to finish
-                if (this->threads[i]->joinable())
-                    this->threads[i]->join();
+                    if (this->threads[i]->joinable())
```

Comment:
why the extra identation?

Reply:
Good question. This is vendor code, so we're leaving its original formatting!